test/device.go

```diff
@@ -23,3 +27,7 @@ func (d *device) SystemScale() float32 {
 func (d *device) SystemScaleForWindow(fyne.Window) float32 {
 	return 1
 }
+
+func (*device) IsBrowser() bool {
+	return (runtime.GOARCH == "js" || runtime.GOOS == "js")
```

Comment:
```suggestion
	return runtime.GOARCH == "js" || runtime.GOOS == "js"
```
Similar nitpick as above. We should try to avoid unnecessary clutter where we can :)